elasticsearch-5.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,165 @@
+registryItems:
+  applications:
+  - {elasticsearch-scenario: nq 0}
+  instances:
+  - {elasticsearch-scenario: 1}
+  operationNames:
+  - elasticsearch-scenario: [Elasticsearch/IndexRequest, Elasticsearch/GetRequest, Elasticsearch/SearchRequest, Elasticsearch/UpdateRequest, Elasticsearch/DeleteRequest, Elasticsearch/DeleteIndexRequest, Elasticsearch/actionGet, /case/elasticsearch]
+  heartbeat: []
+segmentItems:
+- applicationCode: elasticsearch-scenario
+  segmentSize: 1
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: Elasticsearch/IndexRequest
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Database
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 45
+      componentName: ''
+      isError: false
+      spanType: Exit
+      peer: not null
+      peerId: 0
+      tags:
+      - {key: db.type, value: Elasticsearch}
+      - {key: db.instance, value: not null}
+      - {key: db.statement, value: not null}
+      - {key: node.address, value: not null}
+      - {key: es.indices, value: not null}
+      - {key: es.types, value: not null}
+    - operationName: Elasticsearch/actionGet
+      operationId: 0
+      parentSpanId: 0
+      spanId: 2
+      spanLayer: null
```

Comment:
Why  the`spanLayer` is null?

Reply:
This is a LocalSpan and does not set spanLayer for it.